.github/workflows/tests.yml

```diff
@@ -54,4 +54,14 @@ jobs:
     - uses: actions/upload-artifact@v2
       with:
         name: coverage
-        path: htmlcov
\ No newline at end of file
+        path: htmlcov
+  frontend-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: --cwd ./core/frontend install
+      - uses: borales/actions-yarn@v2.3.0
```

Comment:
missing linter step, as we discussed before, we should run linter to make sure that is really running and we do not break the implicit behaviour of linter on build.

Reply:
Done.